backend/src/entities/tags/router.py

```diff
@@ -18,7 +18,7 @@
 @router.get("/search/{search}")
 def get_all_tags_by_table(
     search: str,
-    tables: List[str] = Body(...),
+    tables: Annotated[list[str] | None, Query()] = None,
```

Comment:
Annotated[list[str] | None, Query()]: This specifies that the tables parameter is of type list[str] or None, and it uses the Query() class from FastAPI to define the parameter's behavior for query parameters in the URL. The Query() class allows you to define additional properties and constraints for the query parameter.
tables: Annotated[list[str] | None, Query()] = None: This sets a default value of None for the tables parameter if it's not provided in the URL. The parameter is expected to be part of the URL path (due to the route definition), and it can be passed as a query parameter as well.
Je vais prendre ta version elle est plus concise et ca évite un import.

Reply:
C'est chat gpt la source ?